src/components/VerificationChain/VerificationPhases.tsx

```diff
@@ -0,0 +1,206 @@
+              return newIndices;
+            });
+
+            // Schedule verification of the next step
+            setTimeout(() => {
+              verifyStep(phase, stepIndex + 1);
+            }, 500); // Small delay before starting the next step
+          }
+          // If this was the last step in the phase
+          else {
+            // Set verifying index to -1 to stop the animation
+            setVerifyingIndices((prev) => {
+              const newIndices = [...prev];
+              newIndices[phase] = -1;
+              return newIndices;
+            });
+
+            // Mark this phase as completed after a small delay
+            setTimeout(() => {
+              setCompletedPhases((prev) => {
+                const newCompleted = [...prev];
+                newCompleted[phase] = true;
+                return newCompleted;
+              });
+
+              // If there are more phases, start the next one
+              if (phase < allSteps.length - 1) {
+                setCurrentPhase(phase + 1);
+
+                // Schedule verification of the first step in the next phase
+                setTimeout(() => {
+                  setVerifyingIndices((prev) => {
+                    const newIndices = [...prev];
+                    newIndices[phase + 1] = 0; // Start next phase
+                    return newIndices;
+                  });
+
+                  verifyStep(phase + 1, 0);
+                }, 1000); // Small delay before starting the next phase
+              }
+            }, 500); // Small delay after the last step is verified
+          }
+        }, 2000); // Wait for the animation to complete before marking as verified
+      };
+
+      // Start the verification process
+      verifyStep(0, 0);
+    }, 500); // Initial delay before starting verification
+
+    // Cleanup function
+    return () => {
+      clearTimeout(initialDelay);
+    };
+  }, []); // Empty dependency array means this runs once on mount
+
+  // Arrow component that changes direction based on screen size
+  const DirectionalArrow = () => (
+    <Arrow
+      responsiveDirection={{
+        mobile: "down",
+        desktop: "right",
+      }}
+      className="text-neutral-300"
+    />
+  );
+
+  return (
+    <div className="pt-2 pb-6 lg:pt-4 lg:pb-8">
+      <div className="max-w-6xl mx-auto">
+        <div className="flex flex-col md:flex-row items-center justify-between">
+          <div className="w-full md:w-[33%]">
+            <PhaseColumn
+              title="DNSSEC Chain"
+              steps={dnssecSteps}
+              result="caltech.edu."
+              showTick={true}
+              isActive={verificationStarted && currentPhase === 0}
+              verifyingIndex={verifyingIndices[0]}
+              verifiedIndices={verifiedSteps[0]}
+              isCompleted={completedPhases[0]}
+            />
+          </div>
+
+          <div>
+            <DirectionalArrow />
+          </div>
```

Comment:
Maybe we should define the `div` inside `DirectionalArrow`? Also, all arrows are directional, so that adjective is redundant.